deltachat-rpc-client/tests/test_something.py

```diff
@@ -389,3 +390,97 @@ def test_qr_setup_contact(acfactory) -> None:
         event = alice.wait_for_event()
         if event["kind"] == "SecurejoinInviterProgress" and event["progress"] == 1000:
             return
+
+
+def test_verified_group_recovery(acfactory, rpc) -> None:
+    ac1, ac2, ac3 = acfactory.get_online_accounts(3)
+
+    logging.info("ac1 creates verified group")
+    chat = ac1.create_group("Verified group", protect=True)
+    assert chat.get_basic_snapshot().is_protected
+
+    logging.info("ac2 joins verified group")
+    qr_code, _svg = chat.get_qr_code()
+    ac2.secure_join(qr_code)
+    while True:
+        event = ac1.wait_for_event()
+        if event.kind == "SecurejoinInviterProgress" and event["progress"] == 1000:
+            break
+
+    logging.info("ac3 joins verified group")
+    ac3_chat = ac3.secure_join(qr_code)
+    while True:
+        event = ac1.wait_for_event()
+        if event.kind == "SecurejoinInviterProgress" and event["progress"] == 1000:
+            break
+
+    logging.info("ac2 logs in on a new device")
+    ac2.stop_io()
+    ac2_clone = acfactory.get_unconfigured_account()
+    for i in ["addr", "mail_pw"]:
+        ac2_clone.set_config(i, ac2.get_config(i))
+    rpc.remove_account(ac2.id)
+    ac2_clone.configure()
+    ac2 = ac2_clone
+    del ac2_clone
+
+    logging.info("ac2 reverifies with ac3")
+    qr_code, _svg = ac3.get_qr_code()
+    ac2.secure_join(qr_code)
+
+    while True:
+        event = ac3.wait_for_event()
+        if event.kind == "SecurejoinInviterProgress" and event["progress"] == 1000:
+            break
```

Comment:
wonder if 
```
ac3.wait_securejoin_completed(ac2.get_config("addr"))
```
would be a nice helper as well, also to not have so many places that deal with ```inviterprogress==1000`.